pyproject.toml

```diff
@@ -44,6 +44,8 @@ mypy = "^1.8.0"
 isort = "^5.13.2"
 flake8 = "^7.0.0"
 beautifulsoup4 = "^4.12.3"
+pytest-repeat = "^0.9.3"
```

Comment:
Is this used? It's the only reference to "repeat" I see in this PR, but not sure if its a peer dependency of something maybe. (spoiler: I lack most basic Python knowledge).

Reply:
More so used for testing locally